src/api/trips_v2.ts

```diff
@@ -21,9 +21,13 @@ export async function tripsSearch(
       place: query.from.place,
     },
     when: query.when,
+    cursor: query.cursor,
   };
 
-  const results = await post<TripsQuery>(url, cleanQuery, opts);
+  const results = await post<TripsQuery>(url, cleanQuery, {
+    ...opts,
+    baseURL: 'http://Grans-MacBook-Pro-2.local:8080',
```

Comment:
Tror vi må finne en bedre måte å override URL, ikke helt heldig
```suggestion
   // baseURL: 'http://Grans-MacBook-Pro-2.local:8080',
```